lib/cartopy/tests/io/test_downloaders.py

```diff
@@ -17,21 +17,21 @@
 
 def test_Downloader_data():
     di = cio.Downloader('https://testing.com/{category}/{name}.zip',
-                        os.path.join('{data_dir}', '{category}',
-                                     'shape.shp'),
-                        '/project/foobar/{category}/sample.shp')
+                        str(Path('{data_dir}') / '{category}' / 'shape.shp'),
+                        str(Path('/project') / 'foobar' / '{category}' /
+                            'sample.shp'))
 
     replacement_dict = {'category': 'example',
                         'name': 'test',
-                        'data_dir': os.path.join('/wibble', 'foo', 'bar')}
+                        'data_dir': str(Path('/wibble') / 'foo' / 'bar')}
 
     assert di.url(replacement_dict) == 'https://testing.com/example/test.zip'
 
     assert (di.target_path(replacement_dict) ==
-            os.path.join('/wibble', 'foo', 'bar', 'example', 'shape.shp'))
+            Path('/wibble') / 'foo' / 'bar' / 'example' / 'shape.shp')
 
     assert (di.pre_downloaded_path(replacement_dict) ==
-            '/project/foobar/example/sample.shp')
+            Path('/project') / 'foobar' / 'example' / 'sample.shp')
```

Comment:
Seems like this could have just stayed as:
```suggestion
            Path('/project/foobar/example/sample.shp'))
```

Reply:
Yeah, I was confused about whether a windows system would replace those with path separators or not... Or if we even want it to in this case.

On windows is `Path("/foo/bar")` equivalent to `Path("/foo") / "bar"` ?

Reply:
`Path` normalizes to system directory separator and those are both equivalent. `PosixPath` always uses `/`. But in any case, Windows accepts both `/` and `\`, so it doesn't matter.

Reply:
Good to know, thank you!